src/services/YotiService.ts

```diff
@@ -55,43 +58,15 @@ export class YotiService {
 			.toString("base64");
 	}
 
-
-	//TODO: Hardcoded now, will update with values from Person Identity table once /session work completed
-	private getStructuredPostalAddress(): StructuredPostalAddress {
-		return {
-			address_format: 1,
-			building_number: "74",
-			address_line1: "AddressLine1",
-			town_city: "CityName",
-			postal_code: "E143RN",
-			country_iso: "GBR",
-			country: "United Kingdom",
-		};
-	}
-
 	private getApplicantProfile(
 		personDetails: PersonIdentity,
 	): ApplicantProfile {
-		const givenNames: string[] = [];
-		const familyNames: string[] = [];
-
-		for (const name of personDetails.names) {
-			const nameParts = name.nameParts;
-			for (const namePart of nameParts) {
-				if (namePart.type === "GivenName") {
-					givenNames.push(namePart.value);
-				}
-				if (namePart.type === "FamilyName") {
-					familyNames.push(namePart.value);
-				}
-			}
-		}
+		const nameParts = personIdentityUtils.getNames(personDetails);
 
 		return {
-			full_name: `${givenNames[0]} ${familyNames}`,
+			full_name: `${nameParts.givenNames[0]} ${nameParts.familyNames[0]}`,
 			date_of_birth: `${personDetails.birthDates.map((bd) => ({ value: bd.value }))[0].value}`,
-			structured_postal_address:
-				this.getStructuredPostalAddress(),
+			structured_postal_address: personIdentityUtils.getYotiStructuredPostalAddress(personDetails),
```

Comment:
Dont we require the emailAddress also to be returned?

Reply:
Not here, only when generating the PDF https://github.com/alphagov/di-ipv-cri-f2f-api/pull/39/files/f2e79b1e54d3601d7a4d7266415477c91aaa1d60#diff-111f7d52cf71bdd4103a5303883d6cffdcd013c1c7a1245d33a14d39cf36a583R215